grammar.js

```diff
@@ -356,8 +359,15 @@ module.exports = grammar({
       $.ffi_identifier,
       $.this,
       $.none,
+      $.error,
+      $.compile_intrinsic,
+      $.compile_error,
     ),
 
+    error: _ => 'error',
+    compile_intrinsic: $ => 'compile_intrinsic',
+    compile_error: $ => prec.left(seq('compile_error', optional($.string))),
+
```

Comment:
good catch! I didn't notice these in any of the core ponyc files though

Reply:
Here is a good example for `compile_intrinsic`: https://stdlib.ponylang.io/src/builtin/pointer/
Here is one for `error`: https://stdlib.ponylang.io/src/builtin/array/#L-0-236
This is a good one for `compile_error`: https://stdlib.ponylang.io/src/signals/sig/

Reply:
Awesome, thanks!